Makefile

```diff
@@ -76,7 +76,7 @@ serve:
 	node server.js
 
 test:
-	./node_modules/.bin/jasmine-node test/katex-spec.js
+	./node_modules/.bin/jasmine-node test/*-spec.js
```

Comment:
As per #387, it's only jasmine-node which is broken and unmaintained, not jasmine itself. Running tests in browser might be easier with jasmine as well. So the need to switch doesn't seem as urgent now as it did at first.